src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheckExamplesTest.java

```diff
@@ -33,28 +34,45 @@ protected String getPackageLocation() {
 
     @Test
     public void testExample1() throws Exception {
+        final String configFilePath = getPath("Example1.java");
+        final String propertyFilePath = getPath("Example1.properties");
```

Comment:
You should have few files, as translation files are usually in different files, not a single file .
Do array of files, to let `verifyWithInlineConfigParserSeparateConfigAndTarget` run on array (few files)